packages/web-components/fast-components/src/radio-group/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-radio-group
+An implementation of a [radio](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) as a form-connected web-component.
+
```

Comment:
Copy/paste? Can we replace this with a link pointing to the W3C specification as there is no standard element for radiogroup?

Reply:
sometimes the link also describes radio group but I'll double check this one and the others

Reply:
fixed, it wasn't a copy/paste error as that link described mostly radio-group, but I changed it to be the w3c description of radio-group